example/from_json/movie_sample.dart

```diff
@@ -13,54 +50,44 @@ main() {
   // Define schema in code
   //////////////////////////////////////////////////////////////////////
   var movieSchema = {
-    "title" : "movie data",
-    "additionalProperties" : false,
-    "required" : [ "movies" ],
-    "properties" : {
-      "movies" : { r"$ref" : "#/definitions/movie_map" }
+    "title": "movie data",
+    "additionalProperties": false,
+    "required": ["movies"],
+    "properties": {
+      "movies": {r"$ref": "#/definitions/movie_map"}
     },
-    "definitions" : {
-      "movie" : {
+    "definitions": {
+      "movie": {
         "additionalProperties": false,
-        "required" : [ "title", "year_made", "rating" ],
+        "required": ["title", "year_made", "rating"],
         "properties": {
-          "title" : { "type" : "string" },
-          "year_made" : { "type" : "integer" },
-          "rating" : { "type" : "integer" }
+          "title": {"type": "string"},
+          "year_made": {"type": "integer"},
+          "rating": {"type": "integer"}
         }
       },
-      "movie_map" : {
+      "movie_map": {
         "type": "object",
-        "additionalProperties": { r"$ref": "#/definitions/movie" },
+        "additionalProperties": {r"$ref": "#/definitions/movie"},
         "default": {}
       }
     }
   };
 
   var movies = {
-    "movies" : {
-      "the mission" : {
-        "title":"The Mission",
-        "year_made":1986,
-        "rating":5
-      },
-      "troll 2" : {
-        "title":"Troll 2",
-        "year_made":1990,
-        "rating":2
-      }
+    "movies": {
+      "the mission": {"title": "The Mission", "year_made": 1986, "rating": 5},
+      "troll 2": {"title": "Troll 2", "year_made": 1990, "rating": 2}
     }
   };
 
-  Schema.createSchema(movieSchema)
-    .then((schema) {
-      var validator = new Validator(schema);
-      bool validates = validator.validate(movies);
-      if(!validates) {
-        print("Errors: ${validator.errors}");
-      } else {
-        print('$movies:\nvalidates!');
-      }
-    });
-
+  Schema.createSchema(movieSchema).then((schema) {
+    var validator = new Validator(schema);
+    bool validates = validator.validate(movies);
+    if (!validates) {
+      print("Errors: ${validator.errors}");
```

Comment:
I would imagine we'd want to change these prints to go through the logger, but that can be saved for another time.

Reply:
might do it if the smithy setup takes a while anyway

Reply:
saving this for a logging cleanup ticket: EPL-6664